### ISSUE TITLE
Menuhax freezes at yellow screen

### ISSUE THREAD

**Issue by tranxuanthang** [yellows8/3ds_homemenuhax#46]
I'm using a 9.0.0-20U o3ds. It happens randomly.

![c360_2015-11-20-20-00-09-010](https://cloud.githubusercontent.com/assets/15942946/11300916/57d84968-8fc5-11e5-9f2b-6a6a6b0909fe.jpg)

**Comment by yellows8** [comment]
That's an issue with *hax payload, no idea why it happens so often only for certain users, unless it's system-version related(myself not included when I last tested it on Old3DS/New3DS v10.1 a while ago).

**Comment by Cydget** [comment]
Wait a second. Did you just say that menu hax works on your 10.2 3ds?

**Comment by yellows8** [comment]
Oh oops, typo....

**Comment by Pierro21** [comment]
Hi, I have the same issue with my n3ds 9.0.0-20E except that it's at the white screen.

**Comment by Pierro21** [comment]
I think it's happens 1 for 2 times when I boot...

**Comment by ghost** [comment]
FWIW, out of the dozen or so 3DS I applied HomemenuHax to, it really seems to only rear it's head on the 9.0-9.2 series. (been okay on the others so far)

2DS/N3DS seem to be very rare to freeze on yellow screen, but most 9.0.0-20 and 9.2.0-20 seem to be 50% boot rate with yellow screen. 

Edit: Meant Old3DS with 9.x seem to have the most trouble.

**Comment by yellows8** [comment]
Yeah, Old3DS v9.0-v9.2 seems to be the worst(I hardly test menuhax with that).

**Comment by Victorma** [comment]
Sorry for re-opening this, I found this on reddit (maybe you've seen it already but anyways) https://www.reddit.com/r/3dshacks/comments/413mtd/is_there_a_way_to_improve_the_boot_rate_of/ and it worked really well for me (O3DS 9.2.0-20) with a 100% rate of booting and even faster booting times than with standard homemenuhax. I hope it helps you to fix the issue :D (since homemenuhax you say is based on themehax)

**Comment by yellows8** [comment]
@Victormafire I don't see how that's supposed to work at all, pre-v2.0 will fail at trying to process the *hax payload(and even if it did work, you'd really only be affecting the *hax payload ......).

**Comment by jon2491** [comment]
@yellows8 @Victormafire well, that worked for me, using themehax payloads on 9.2 (loading the themehax payload with menuhax) I'm having around 95% boot rate (I've been using it for more than a week).

**Comment by ghost** [comment]
Happens to me randomly too JPN N3DS 9.1

{Posts merged by yellows8}

Sometimes it's yellow most of the time it's white

**Comment by s3r1alpari4h** [comment]
Sometimes it's yello most of the time it's white

**Comment by s3r1alpari4h** [comment]
Yellow*

**Comment by My1** [comment]
I reboot this one, same issue. my screen freezes from time to time on yellow
o3DSXL 11.0E menuhax 3.1
